addons/survey/views/survey_templates.xml

```diff
@@ -145,6 +145,7 @@
                     };
                 </script>
                 <t t-call-assets="web.assets_common"/>
+                <t t-call-assets="survey.minimal_assets"/>
```

Comment:
Why? :)

Reply:
Actually this is a bug since 12.1 but I asked for a fix that will be ported soon, it was done at https://github.com/odoo/odoo/pull/33218 so this one will be removed from this PR once ported and rebased.
But meanwhile this fix is needed on the PR for a green runbot otherwise since now tests are executed in debug mode tests (and thus debug mode), the error about duplicated files is not catch anymore, making the tour to fail.